src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/matchxpath/Example4.java

```diff
@@ -0,0 +1,24 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MatchXpath">
+      <property name="query"
+       value="//VARIABLE_DEF[./ASSIGN/EXPR/LITERAL_NEW and
+       not(./TYPE/IDENT[@text='var'])]"/>
```

Comment:
`and` in always on new line when wrapping

Reply:
Done.